pkg/persistence/account/loader/load_test.go

```diff
@@ -145,11 +153,21 @@ func TestValidateReferences(t *testing.T) {
 			expected:       ErrIdFieldMissing,
 			expectedErrMsg: `error validating account resources: no ref id field found`,
 		},
+		{
+			name:     "mixing configs and account resources",
+			path:     "testdata/configs-accounts-mixed.yaml",
+			expected: ErrMixingConfigs,
+		},
 		{
 			name:     "valid",
 			path:     "testdata/valid.yaml",
 			expected: nil,
 		},
+		{
+			name:     "valid loading configs",
```

Comment:
test this explitily please like the test on line 70 - then it's clear from looking at the test, that not only do we want no error, but also 0 groups, policies and users